app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -150,6 +150,10 @@ const InnerServiceDiscoveryResults = ({
     sortAlgoliaSubcategoryRefinements,
   } = category;
 
+  const escapeApostrophes = (string: string) => {
+    return string.replace(/'/g, "\\'");
+  };
```

Comment:
How about using terser syntax since this is a simple function?
```suggestion
  const escapeApostrophes = (str: string): string => str.replace(/'/g, "\\'");
```